src/lib/pages/validator-details/components/validator-overview/VotingPowerOverview.tsx

```diff
@@ -1,21 +1,32 @@
-import { Flex, Heading, Text } from "@chakra-ui/react";
+import { Flex, Grid, Heading, Text } from "@chakra-ui/react";
+import type { BigSource } from "big.js";
+import type Big from "big.js";
 
+import { TokenImageRender } from "lib/components/token";
 import { ValueWithIcon } from "lib/components/ValueWithIcon";
-
-// TODO remove mockup data
+import { getUndefinedTokenIcon } from "lib/pages/pools/utils";
+import type { AssetInfos, Option, Token, U, USD } from "lib/types";
+import {
+  calculateAssetValue,
+  divWithDefault,
+  formatPrettyPercent,
+  formatPrice,
+  formatUTokenWithPrecision,
+  toToken,
+} from "lib/utils";
 
 const VotingPowerDetail = ({
   label,
   percent,
   amount,
-  denom,
+  denom = "",
```

Comment:
```
symbol,
denom,
```

Reply:
Will not pass `symbol` as prop, since it already contained inside `assetInfo`